### ISSUE TITLE
Support more keywords (Postgres)

### ISSUE THREAD

**Issue by bernardoamorim7** [mtxr/vscode-sqltools#1325]
**Is your feature request related to a problem? Please describe.**

I have a difficult time using the extension with Postgres because some keywords aren't highlighted as they should, and it's especially difficult reading procedures sometimes because of that.

**Describe the solution you'd like**

Add more keywords to the regexp that looks for keywords in `language/sql.tmLanguage.json`

The keywords I have right now are:
`elsif|raise|notice|exception|warning|conflict|do|nothing|default|varying|add|column|tablespace|temporary|for|each|index|trigger|rename`

Adding this to the end of the regexp fixes the issue for my use case.

**Describe alternatives you've considered**

I have no other alternative as of right now, perhaps there's a better option I can't think of?

**Comment by xpz24** [comment]
Yeah, I faced the same issue too, I assumed those keywords would be supported out of the box.

**Comment by xpz24** [comment]
As @bernardoamorim7 mentioned, the best workaround is to add the keywords directly to the regex expression in **"keyword.other.sql"** rule.

I also replaced the sql.tmLanguage.json and language-configuration.json of extension from the ones in vscode repo.
**Highlighting works out of the box** 
default
add
for
index

**Works once added to the "keyword.other.sql" rule**
conflict
do
each
elsif
exception
nothing
notice
raise
rename
temporary
warning

**varying**
Highlights in `character varying(100)`

**column**
Highlights `in comment on column {column_name} is`

**Had modifying the rule "meta.drop.sql" and deleting the entry below it**
trigger
tablespace

**Worked when typing:** `CREATE TABLSPACE tablespace1`
**Only "table" of tablespace was highlighted:** `DROP TABLESPACE tablespace1` 

**Original**
```
{
    "captures": {
        "1": { "name": "keyword.other.create.sql" },
        "2": { "name": "keyword.other.sql" }
    },
    "match": "(?i:^\\s*(drop)\\s+(aggregate|conversion|database|domain|function|group|index|language|operator class|operator|rule|schema|sequence|table|tablespace|trigger|type|user|view))",
    "name": "meta.drop.sql"
},
{
    "captures": {
        "1": { "name": "keyword.other.create.sql" },
        "2": { "name": "keyword.other.table.sql" },
        "3": { "name": "entity.name.function.sql" },
        "4": { "name": "keyword.other.cascade.sql" }
    },
    "match": "(?i:\\s*(drop)\\s+(table)\\s+(\\w+)(\\s+cascade)?\\b)",
    "name": "meta.drop.sql"
},
```
Not the best way to write the rule as cascade should not work with all these keywords. But for some reason the original rule implementation was not working at all.
**Modification**
```
{
    "captures": {
        "1": { "name": "keyword.other.create.sql" },
        "2": { "name": "keyword.other.sql" },
        "4": { "name": "entity.name.function.sql" },
        "5": { "name": "keyword.other.cascade.sql" }
    },
    "match": "(?i:^\\s*(drop)\\s+(aggregate|conversion|database|domain|function|group|index|language|operator class|operator|rule|schema|sequence|table|tablespace|trigger|type|user|view)\\s+)(['\"`]?)(\\w+)\\3(\\s+(?i:cascade))?",
    "name": "meta.drop.sql"
},
```
